posthog/async_migrations/runner.py

```diff
@@ -81,6 +81,11 @@ def start_async_migration(migration_name: str, ignore_posthog_version=False) ->
         process_error(migration_instance, f"Migration precheck failed with error:{error}")
         return False
 
+    ok, error = run_migration_healthcheck(migration_instance)
+    if not ok:
+        process_error(migration_instance, f"Migration healthcheck failed with error:{error}")
```

Comment:
you're gonna have a merge conflict with the startup status change PR (i.e. we're missing the status here to be `failedAtStatup`). Feel free to handle in either depending on which order you land them.